pkg/R/RDD.R

```diff
@@ -70,10 +75,12 @@ setMethod("initialize", "PipelinedRDD", function(.Object, prev, func, jrdd_val)
       func(split, prev@func(split, iterator))
     }
     .Object@func <- pipelinedFunc
+    .Object@funcAccum <- cloneAccumulator(prev@funcAccum)
```

Comment:
So if we are going to go down this path of having a list of functions rather then we can get rid of the `pipelinedFunc` block above this.

However this brought up one more idea -- Do we really need a list of functions or can we somehow assign the environment of `pipelinedFunc` with something like `environment(pipelinedFunc) <- environment(prev@func)` ?
Basically what I am thinking about is that if we can do the pipelining on the worker side, is it possible to an equivalent pipeline on the driver side here.